docs/settings/security-settings.asciidoc

```diff
@@ -370,7 +358,31 @@ xpack.security.audit.appender:
 <1> Rotates log files every 24 hours.
 <2> Keeps maximum of 10 log files before deleting older ones.
 
-| `xpack.security.audit.appender.type`
+[IMPORTANT]
+============
+{ess} does not support custom log file policies. To enable audit logging on {ess} only specify:
+
+[source,yaml]
+----------------------------------------
+xpack.security.audit.enabled: true
+xpack.security.audit.appender.type: rolling-file
+----------------------------------------
+============
+
+[WARNING]
+============
+To enable the legacy audit logger (**deprecated**) only specify:
```

Comment:
I'm not 100% sure how this will look until it's rendered, but I think we should use the "deprecated" keyword here:
```suggestion
deprecated:[7.15.0,"In 8.0 and later, the legacy audit logger will be removed, and this setting alone will instead enable the ECS audit logger with a default appender."] To enable the legacy audit logger, only specify:
```

It should render as a bit of crossed out text that shows a tooltip when you mouse over; here's an example of what it will look like:

![image](https://user-images.githubusercontent.com/5295965/124815942-e39e5f80-df35-11eb-8d1f-4fa00d822ce7.png)